week4/views/show.ejs

```diff
@@ -0,0 +1,49 @@
+<%- include('./partials/header') %>
+<div class="container border mt-3 p-3">
+
+  <h3><%=cohort.name%></h3>
+  <p> <%=cohort.members%> </p>
+  <hr>
+  <form action="/cohorts/<%= cohort.id %>/form" method = "get">
```

Comment:
should be a `GET` to `/cohorts/:id`, not to this custom route. as defined in the homework.